service/src/main/java/uk/nhs/adaptors/oneoneone/cda/report/mapper/AddressMapper.java

```diff
@@ -21,11 +25,15 @@ public Address mapAddress(AD itkAddress) {
         Address address = new Address();
 
         Arrays.stream(itkAddress.getStreetAddressLineArray())
-                .map(nodeUtil::getNodeValueString)
-                .forEach(address::addLine);
+            .map(nodeUtil::getNodeValueString)
+            .forEach(address::addLine);
 
         if (itkAddress.isSetUse()) {
-            address.setUse(AddressUse.fromCode(getAddressUseString(itkAddress.getUse().get(0).toString())));
+            if (itkAddress.getUse().get(0).toString().equals("PHYS")) {
+                address.setType(AddressType.fromCode(getAddressUseString(itkAddress.getUse().get(0).toString())));
```

Comment:
let's not use getAddressUseString method here - we don't want to mix use and type mapping in one method